autoload/intero/repl.vim

```diff
@@ -199,6 +199,30 @@ function! g:intero#repl#get_type_signature_line_replacement(existing_line, type_
                 \ + l:indented + [l:indent . l:suffix]
 endfunction
 
+let s:word_under_cursor = ""
+function! intero#repl#type_on_hover()
+    if g:intero_started
+        let l:new_word_under_cursor = expand("<cword>")
+        if s:word_under_cursor !=# l:new_word_under_cursor
+            let l:ident = intero#util#get_haskell_identifier()
+            if !empty(l:ident)
+                call intero#process#add_handler(function('intero#repl#type_on_hover_handler'))
+                call intero#repl#send(':type ' . l:ident)
```

Comment:
Not sure, maybe someone else can verify, but could this instead using `:type-at`?

``` vim
call intero#repl#send(intero#util#make_command(':type-at'))
```

Reply:
That works too, I've changed to that in an upcoming commit 👍

Reply:
Nice!

Reply:
Ah, wait, I remember why I did it this way: with `call intero#repl#send(intero#util#make_command(':type-at'))` you get `Couldn't guess the module name. Is this module loaded?` if the module hasn't been loaded into the repl (or something like that), but with `call intero#repl#send(':type ' . l:ident)` I can circumvent that.